content/how-to/dedicated-hosting/how-to-request-a-csr/index.md

```diff
@@ -1,41 +1,43 @@
 ---
 permalink: how-to-request-a-csr
 audit_date:
-title: 'How to request a CSR'
+title: 'How to Request a CSR'
 type: article
 created_date: '2021-04-15'
 created_by: Alexandra Soares
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2023-02-01'
```

Comment:
Update the date if you need to make other changes. If not then I will change it right before merge.